include/godot_cpp/classes/wrapped.hpp

```diff
@@ -109,6 +123,11 @@ class Wrapped {
 	GodotObject *_owner = nullptr;
 };
 
+template <typename T, std::enable_if_t<std::is_base_of<::godot::Wrapped, T>::value, bool>>
+_ALWAYS_INLINE_ void _pre_initialize() {
```

Comment:
Actually, this recent change doesn't look quite right - shouldn't this have the `= true`?

Reply:
Add `= true` at here can't compile successfully because of the redefinition of template default argument. Previous default template argument defined in the forward declaration.

My compiler is clang.

Reply:
Hm, interesting. So long as the right version of this template function is selected in all compilers, I guess it's fine. The automated tests should confirm if it's working or not, but I think CI only runs the tests with a GCC build.

Reply:
I add this check in `Wrapped` constructor:
```C++
	if (likely(_constructing_class_binding_callbacks)) {
		godot::internal::gdextension_interface_object_set_instance_binding(_owner, godot::internal::token, this, _constructing_class_binding_callbacks);
		_constructing_class_binding_callbacks = nullptr;
	} else {
		ERR_PRINT("BUG: create a Godot Object without binding callbacks.");
		CRASH_NOW_MSG("BUG: create a Godot Object without binding callbacks.");
	}
```

If it can not select the correct `_pre_initialize` function to create an Godot object (both natives and extensions), the application will print an error and crash directly.

Reply:
Ok, that will certainly prevent problems with this sneaking by unnoticed :-)